release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -472,6 +472,43 @@ For more information about disabling the Image Registry Operator, see xref:../in
 [id="ocp-4-14-olm"]
 === Operator lifecycle
 
+[id="ocp-4-14-olmv1"]
+==== {olmv1-first} (Technology Preview)
+
+Operator Lifecycle Manager (OLM) has been included with {product-title} 4 since its initial release. {product-title} {product-version} introduces components for a next-generation iteration of OLM as a Technology Preview feature, known during this phase as _{olmv1}_. This updated framework evolves many of the concepts that have been part of previous versions of OLM and adds new capabilities.
+
+During this Technology Preview phase of {olmv1} in {product-title} 4.14, administrators can explore the following features:
+
+Fully declarative model that supports GitOps workflows::
+{olmv1} simplifies Operator management through two key APIs:
++
+--
+* The `Operator` API, provided by the new Operator Controller component, streamlines management of installed Operators by consolidating user-facing APIs into a single object. This empowers administrators and SREs to automate processes and define desired states by using GitOps principles.
```

Comment:
Actually, OLM has the `Operator` kind before, which is from the CRD `operators.operators.coreos.com`. Here is the new `Operator` kind from CRD: `operators.operators.operatorframework.io`. Both of them use the `operator` strings for now.
See the discussion on Slack: https://redhat-internal.slack.com/archives/GHMALGJV6/p1689573773633059 or the https://issues.redhat.com/browse/OCPBUGS-22094
```console
MacBook-Pro:~ jianzhang$ oc get operators.operators.operatorframework.io 
NAME           AGE
quay-example   4h10m
MacBook-Pro:~ jianzhang$ oc get operators.operators.coreos.com 
NAME                                                AGE
cluster-logging.openshift-logging                   7h13m
elasticsearch-operator.openshift-operators-redhat   7h13m
loki-operator.openshift-operators-redhat            7h13m
oadp-operator.e2e-test-olm-a-fwobi02w-gsc8h         3h42m
```
So, to avoid confusion, here I'd suggest using the `operators.operators.operatorframework.io` instead of `operator`.

Reply:
Good point! I had created this [olmv1-operator-api-group.adoc](https://github.com/openshift/openshift-docs/pull/65963/files#diff-a81ceaf5428923a809911af7ed6b606a17e60003ee16e3926cf87c64384ddc55) `[NOTE]` snippet in another WIP PR, which talks about that difference, so I'll re-use that here as well.